modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -61,8 +61,11 @@ You can publish the API fragment to Anypoint Exchange. Click the icon that is in
 
 image::publish-to-exchange.png[title="Publish to Exchange icon",46,52,align="left"]
 
-The fragment can then be included as a dependency in an API-specification project. After revising a fragment, update API specifications that use the fragment to include the new version.
+The fragment can then be included as a dependency in an API-specification project. After revising a fragment, update API specifications that use the fragment to include the new version. +
+
+When you publish the API Fragment to Exchange it creates automatically its API Portal. See "Exchange Public Portals" linked in the *See also* section at the end of this topic.
```

Comment:
"...to Exchange it creates automatically its API Portal" should be:
"...to Exchange, it automatically creates its API Portal"